crates/biome_js_analyze/tests/specs/nursery/noUselessUndefined/invalid.json.snap

```diff
@@ -0,0 +1,814 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid.json
+---
+# Input
+```cjs
+let a = undefined;
+```
+
+# Diagnostics
+```
+invalid.json:1:9 lint/nursery/noUselessUndefined ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Don't use unnecessary undefined.
+  
+  > 1 │ let a = undefined;
+      │         ^^^^^^^^^
+  
+  i undefined is the default value for new variables, parameters, return statements, etc… so specifying it doesn't make any difference.
```

Comment:
These cases don't have a code action. This means they should provide a third note/message that tells the user how to fix the issue.